account_invoice_check_identical_invoice/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2017 - Today Coop IT Easy SCRLfs (<http://www.coopiteasy.be>)
+# - Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Account Invoice Check Identical Invoice",
+    "version": "12.0.1.0.0",
+    "depends": ["base", "account",],
+    "author": "Coop IT Easy SCRLfs",
+    "category": "Accounting & Finance",
+    "website": "https://coopiteasy.be",
+    "description": """
+        This module requires to check this box to validate the invoice 
+        if invoices with the same partner, invoice date and totam alount already
```

Comment:
Move description to DESCRIPTION.rst and add key summary in the __manifest__. You can install pre-commit for theses checks.

Also here, `totam alount` should be `total amount`

Reply:
done

Reply:
sorry, that was misleading. I'll setup pre-commit here next time my compulsive disorders hit.